cfme/configure/tasks.py

```diff
@@ -254,11 +75,185 @@ def is_displayed(self):
             self.tabs.allothertasks.is_displayed)
 
 
-class Tasks(Navigatable):
-    pass
+@attr.s
+class Task(BaseEntity):
+    """Model of an tasks in cfme.
+
+    Args:
+        name: Name of the task.
+        user: User who initiated the task.
+        server: Cfme server.
+        tab: Tab where current task located.
+
+    """
+    OK = 'ok'
+    ERROR = 'error'
+    IN_PROGRESS = 'in_progress'
+
+    name = attr.ib()
+    user = attr.ib(default='admin')
+    server = attr.ib(default='EVM')    # >= 5.9 only
```

Comment:
This needs to be version picked, and possibly changed to 'server_name' as to not confuse it with a `Server` object. How does this behave when testing 58z?